legacy/common.ts

```diff
@@ -1,6 +1,7 @@
 import * as graphlib from '@snyk/graphlib';
 import { DepGraph } from '@snyk/dep-graph';
 
+export { DepGraph };
```

Comment:
```suggestion
export { DepGraph } from '@snyk/dep-graph';
```

Reply:
Using on this line https://github.com/snyk/snyk-cli-interface/blob/aceba28f02ccb592e39c10c7da35c891ac1b2d7f/legacy/common.ts#L37

Reply:
Can we verify this?

Reply:
What do you mean by verify?

Reply:
as in verifying what Or said

Reply:
I verified it